src/main/java/org/folio/rmapi/model/CustomerResources.java

```diff
@@ -0,0 +1,160 @@
+package org.folio.rmapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CustomerResources {
+
+  @JsonProperty("titleId")
+  private String titleId;
```

Comment:
If this model is derived from the documentation mentioned above, I see that in sample response - titleId is an integer and not a string?